app/controllers/teachers/units_controller.rb

```diff
@@ -8,8 +8,7 @@ def create
   end
 
   def index
```

Comment:
just removed the commented out section, changed `classrooms` to `classrooms_i_teach`